CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to [Earthly](https://github.com/earthly/earthly) will be doc
 
 ## Unreleased
 
+### Added
+
+- Earthly now provides the following [builtin ARGs](https://docs.earthly.dev/docs/earthfile/builtin-args): `EARTHLY_VERSION` and `EARTHLY_BUILD_SHA` [#1452](https://github.com/earthly/earthly/issues/1452).
```

Comment:
very nice :tada: (I always forget to update this)

Reply:
This might need to be updated to say that this needs to be enabled with a certain feature flag.